src/snowflake/telemetry/_internal/exporter/otlp/proto/logs/__init__.py

```diff
@@ -67,11 +66,7 @@ def export(self, batch: typing.Sequence[_logs.LogData]) -> export.LogExportResul
 
     @staticmethod
     def _serialize_logs_data(batch: typing.Sequence[_logs.LogData]) -> bytes:
-        # pylint gets confused by protobuf-generated code, that's why we must
-        # disable the no-member check below.
-        return LogsData(
-            resource_logs=encode_logs(batch).resource_logs # pylint: disable=no-member
-        ).SerializeToString()
+        return encode_logs(batch)
```

Comment:
Let's rename this API since we do different thing from what opentelemetry is doing.